package.json

```diff
@@ -12,6 +12,7 @@
   ],
   "sideEffects": false,
   "scripts": {
+    "build-prep": "sed -i '' \"s/export const version: .*$/export const version = '$npm_package_version'/\" build/build.ts",
```

Comment:
My bad for not suggesting this alternative earlier 😅 😅 we don't need the `sed` magic to build the `build/build.ts` file. We can just do: 
```
import { version } from '../../../package.json'
```
whenever we have 
```
import { version } from '../../../build/build.ts'
```

Reply:
`import { version} from ./package.json` will import the entire package.json into the webpack build, though

Reply:
Oh! I see, then probably we have to go with `sed` magic 👍